src/constants.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
```

Comment:
Move all these constants into an array so we can iterate over the array during the shell script creation, to prevent us from forgetting to add new ones to the creation file. 

Then update all the scripts that use the constants by name. Maybe use a class definition so these constants can be loaded using constants.TARGET_ORG for example?